src/storage/db_interface_frontend.py

```diff
@@ -162,7 +163,11 @@ def _convert_result_list_to_dict(search_results):
     # --- statistics
 
     def get_X_last_added_firmwares(self, limit_x=10):
```

Comment:
can this be renamed to "get_last_added_firmware(self, limit)" looks odd right now. And pycharm isn't happy either.